README.md

```diff
@@ -1648,6 +1650,108 @@ console.log(await client.capitalDepositAddress({ coin: 'NEO' }))
 
 ### Futures Authenticated REST endpoints
 
+#### futuresGetOrder
+
+Check an order's status.
+
+##### Weight: 1
+
+- These orders will not be found
+  - order status is CANCELED or EXPIRED, <b>AND</b>
+  - order has NO filled trade, <b>AND</b>
+  - created time + 7 days < current time
+  
+##### Parameters:
```

Comment:
Same goes for all these actually, we're not using that so I'd rather keep it consistent